packages/services/f-braze-adapter/src/services/BrazeConsumer.js

```diff
@@ -0,0 +1,147 @@
+import uniq from 'lodash.uniq';
+import {
+    filterByBrands,
+    filterByEnabledCardTypes,
+    pipe,
+    filterByCurrentlyActive,
+    sortByCardOrder
+} from './utils/index';
+import InvalidConsumerConfigError from './errors/InvalidConsumerConfigError';
+import { LogService } from './logging/logging.service';
+
+class BrazeConsumer {
+    /**
+     * Setter that is called when cards are set on the class, applies any filter methods on those cards and
+     * sets the cards variable
+     * @param cards
+     */
+    set cards (cards) {
+        this._cards = this.applyFilterMethods(cards);
+    }
+
+    constructor ({
+        enabledCardTypes,
+        brands = [],
+        callbacks,
```

Comment:
Callbacks should default to an empty array.

Reply:
default empty object